### ISSUE TITLE
[Suggestion] Choice to Disable in App

### ISSUE THREAD

**Issue by Catbodi** [dmarcotte/easy-move-resize#39]
I love Easy Move Resize and use it on all my Mac's and consider it an absolute necessity. So first I just wanted to thank you for the time you put forward to make something like this. I have a suggestion for the possibility if you ever decide to implement.

- Ability to disable in certain apps

Apps like Photoshop and others where the same trigger key is used results in issues of not being detected. It'd be great if we could select apps to disable it.

**Comment by ryandoyle** [comment]
I was after this too as I have command+click defined as "go to definition" in my various IDEs. I modified the `CGEventTapCallBack`, `myCGEventCallback` to always return the event so its passed through to the application. This works well, I can resize and move and still command-click in my IDEs.

I changed `EMRAppDelegate.m:224` to read

```
return event;
```
instead of 
```
return NULL;
```

I'm not an Objective C or MacOS developer so I'm not sure if this is correct but if it is, maybe this could be implemented?

Referencing #27 too.
